README.md

```diff
@@ -16,6 +16,9 @@ If running Ubuntu 20.04 or other distros based on it you must install this dep a
 You must also tell the installer you are running it as well. 
 
 ## Other Deps
+For Ubuntu 16.04 and greater:
+
+`sudo apt install curl md5sum cmake pkg-config git libao-dev libasound2-dev libavcodec-dev libavformat-dev libbluetooth-dev libenet-dev libgtk2.0-dev liblzo2-dev libminiupnpc-dev libopenal-dev libpulse-dev libreadline-dev libsfml-dev libsoil-dev libsoundtouch-dev libswscale-dev libusb-1.0-0-dev libwxbase3.0-dev libwxgtk3.0-dev libxext-dev libxrandr-dev portaudio19-dev zlib1g-dev libudev-dev libevdev-dev "libpolarssl-dev|libmbedtls-dev" libcurl4-openssl-dev libegl1-mesa-dev libpng-dev qtbase5-private-dev`
```

Comment:
Im unsure about the use of the md5sum package, as I can't find that package on ArchLinux, and the installer looks to support all distros.

Reply:
I also disagree that we should have a full list of ubuntu deps here as things change and we would need to keep this list curated.

The dolphin wiki probably has more eyes on this kinda thing.

Reply:
Hello jsj1027,
I double checked and was mistaken that md5sum was its own package.

I checked 
`man md5sum` 
in Ubuntu
and saw that it was part of the gnu core utils. 
I also booted into Manjaro 20.0.2
and ran 
`whereis md5sum` 
and got back
`md5sum:/usr/bin/md5sum /usr/share/man/man1/md5sum.1.gz` 

So md5sum should be available on Arch and any recent GNU/Linux distro, but it is not its own package.
My mistake.

Reply:
> I also disagree that we should have a full list of ubuntu deps here as things change and we would need to keep this list curated.
> 
> The dolphin wiki probably has more eyes on this kinda thing.
That is fine, pointing to the wiki as you already do works.